### PR TITLE
Managing custom prices for big amount of products

### PR DESCRIPTION
**Pull request by lenadoc**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _data/sidebars/scos_dev_sidebar.yml [hunk 1]
```diff
@@ -3886,6 +3886,8 @@ entries:
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-warehouse-data.html
           - title: HowTo - Import Packaging Units
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-packaging-units.html
+          - title: 'HowTo: Manage merchant custom prices for products via data import'
```
Comment (jeremyfourna) [inline _data/sidebars/scos_dev_sidebar.yml:3889]: We should not label it as Merchant custom prices but more "HowTo: Manage business unit specific prices via data import"

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: 'HowTo: Manage merchant custom prices for products via data import'
```
Comment (jeremyfourna) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md:2]: Same comment as the file above ☝️

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+last_updated: Feb 04, 2022
+template: howto-guide-template
+---
+
+{% info_block warningBox "B2B only" %}
+
+You can apply the approaches described in this article only if your project follows the B2B model, as the [merchant custom prices](https://docs.spryker.com/docs/scos/user/features/{{site.version}}/merchant-custom-prices-feature-overview.html) is only the B2B feature. Also, keep in mind that these approaches don't work for the Marketplace product offers.
```
Comment (jeremyfourna) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md:9]: About

> Also, keep in mind that these approaches don't work for the Marketplace product offers.

We should also add a `warningBox` to say that it is not compatible with Marketplace at the moment

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+{% endinfo_block %}
+
+Let's assume you have a [default price](https://docs.spryker.com/docs/scos/user/features/202108.0/scheduled-prices-feature-overview.html#price-types) for products, and you have merchants for whom you want to make a certain percentage discount from the default price.
```
Comment (jeremyfourna) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md:13]: Should we not put `{{site.version}}` for the `../202108.0/..` in the URL? If yes, this must be corrected in all URL below

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+4. Then, you can do one of the following:
+- Set specific default prices for this merchant manually.
+That means, if you provide say 20% discount on a product for merchant relation `mr001` and change the default price for the product in the [product price file](https://docs.spryker.com/docs/scos/dev/data-import/{{site.version}}/data-import-categories/catalog-setup/pricing/file-details-product-price.csv.html), you would have to go to the `price-product-merchant-relationship-mr001.csv` file and change the price for this merchant as well. Because changing the general default product price does not automatically change the merchant custom price for this product. Thus, with every change of the default product price, you have to manually change the every merchant custom price of this product.
+This option can work for a relatively small amount of products and merchants you provide the specific prices for. For big amount of this data, to avoid too much manual work, we recommend considering the next option.
+- Create a custom script that would automatically adjust merchant-specific prices upon the default price change.
```
Comment (jeremyfourna) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md:37]: As a side note, we could build a relatively small script that will do that until we cover this problem properly in the Product.
We could have this script under "Spryker Utils" lol

**Review by jeremyfourna — CHANGES_REQUESTED** [review]

File: _data/sidebars/scos_dev_sidebar.yml [hunk 6]
```diff
@@ -3886,6 +3872,8 @@ entries:
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-warehouse-data.html
           - title: HowTo - Import Packaging Units
             url: /docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-import-packaging-units.html
+          - title: HowTo - Manage business unit-specific prices via data import
```
Comment (jeremyfourna) [inline _data/sidebars/scos_dev_sidebar.yml:3875]: Do we have `-` between `business unit-specific prices` or we just need a space there?

**Review by jeremyfourna — APPROVED** [review]
I just left a small comment about a `- instead` of a space otherwise it is all good

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-business-unit-specific-prices-via-data-import.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+This option can work for a relatively small amount of products and merchants you provide the specific prices for. For big amount of this data, to avoid too much manual work, we recommend considering the next option.
+- Create a custom script that would automatically adjust merchant-specific prices upon the default price change.
+In your ERP, you might have the base, or default price, defined along with the percentage of discount for your merchants. The script would handle the relation between the default price and the discount for merchants to define the specific price for the merchants. 
+For example, if the percentage of discount for merchant is 20%, the script would generate the `price-product-merchant-relationship-mr001.csv` file with the prices for this `mr001` merchant. In our example, the price would be the default price minus 20%. 
+This approach is optimal for big amount of data. However, its main drawback is in the fact that you need some development effort and that the script runs and processes data outside of the Spryker platform.
```
Comment (VadymSachenko) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-business-unit-specific-prices-via-data-import.md:40]:
```suggestion
This approach is optimal for big amount of data. However, its main drawback is that you need some development effort and that the script runs and processes data outside of the Spryker platform.
```

**Review by VadymSachenko — APPROVED** [review]
Approved. 
Please consider the added comments

**Pull request merged**